ENIGMAsystem/SHELL/Bridges/OpenGLES/OpenGLHeaders.h

```diff
@@ -0,0 +1,21 @@
+/** Copyright (C) 2010-2013 Josh Ventura, Robert B. Colton, Alasdair Morrison
+***
+*** This file is a part of the ENIGMA Development Environment.
+***
+*** ENIGMA is free software: you can redistribute it and/or modify it under the
+*** terms of the GNU General Public License as published by the Free Software
+*** Foundation, version 3 of the license or any later version.
+***
+*** This application and its source code is distributed AS-IS, WITHOUT ANY
+*** WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+*** FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+*** details.
+***
+*** You should have received a copy of the GNU General Public License along
+*** with this code. If not, see <http://www.gnu.org/licenses/>
+**/
+
+// TGMG: I've replaced your include of the switchboard with this hard-coded job until I find
```

Comment:
lol, you can probably just delete this comment and the copyright header, at this point.

Reply:
Crap, I am so sorry, I just lost track of what I was doing and merged it before doing this. My bad. I'll send another pull.

Reply:
Ok, I've sent #1696.